cmake/util/FindCUDA.cmake

```diff
@@ -37,7 +37,7 @@
 #
 macro(find_cuda use_cuda)
   set(__use_cuda ${use_cuda})
-  if(__use_cuda STREQUAL "ON")
+  if(${__use_cuda})
```

Comment:
For readability, let us do ${USE_CUDA} MATCHES ${IS_TRUE_PATTERN} :)

Reply:
I had not made an IS_TRUE_PATTERN because `if(${VAR})` works in all cases, but I will add the pattern for symmetry.